workspace_host/interface.js

```diff
@@ -392,7 +395,7 @@ async function pushContainerContentsToS3(workspace) {
 
   const workspacePath = path.join(workspacePrefix, `workspace-${workspace.launch_uuid}`);
   const s3Path = `workspace-${workspace.id}-${workspace.version}/current`;
-  const settings = _getWorkspaceSettingsAsync(workspace.id);
+  const settings = await _getWorkspaceSettingsAsync(workspace.id);
```

Comment:
I briefly enabled TypeScript error checking in this file, which is how I discovered this. There are too many other errors to leave it enabled permanently (mostly things with `util.callbackify`/`util.promisify`), but this was an easy one to fix.